src/server.rs

```diff
@@ -592,3 +624,14 @@ pub enum ServerEvent {
     ClientConnected { client_id: ClientId },
     ClientDisconnected { client_id: ClientId, reason: String },
 }
+
+/// Starts replication for a connected client.
+///
+/// This event needs to be sent manually if [`ServerPlugin::replicate_after_connect`] is set to `false`.
+///
+/// You must only startin replication for a specific client up to once per connection.
```

Comment:
It's not what happens in this case from this sentence.
And nothing critical will happen, user will just see a warning. So I would suggest to remove it, it's obvious that you don't need to call it multiple times. So let's remove the sentence.
```suggestion
```